src/documents/IoCSS.md

```diff
@@ -47,74 +52,78 @@ fluid.uiOptions(".my-uio-container", {
-|record|(Mutually exclusive with source) A record of options to set at the target.|
-|removeSource|(Only possible if `source` is used) true/false: If true, the `source` options block is removed from its original site in the options structure when it is forwarded to the `target`.|
-|exclusions|(Only possible if `source` is used) A list of EL paths into the `source` material which should not be forwarded. Whether or not `removeSource` is used, these will be retained in their original position in the source component's options.|
-
+|`target`|(Required) An [IoC expression](IoCReferences.md) describing the location in the component tree where the options are to be set. The "context" part of this expression will usually consist of an **IoCSS selector** (see below for format). However, it may also in specialised cases consist of a standard "upwards" IoC context expression, indicating that options are to be distributed to a parent component (this is only meaningful in the case the parent has not finished instantiating).|
+|`record`|(Mutually exclusive with `source`) A record of options to set at the target. This will be interpreted literally - however it will be [expanded](ExpansionOfComponentOptions.md) according to standard options expansion rules. General configuration will be expanded in the context of the component holding `distributeOptions` - however, configuration for listeners and invokers will be expanded at the target component.|
+|`source`|(Mutually exclusive with `record`) An [IoC expression](IoCReferences.md) into the options structure of the source component, referencing what to copy to the target.|
+|&nbsp;&nbsp;`removeSource`|(Only possible if `source` is used) true/false: If true, the `source` options block is removed from its original site in the options structure when it is forwarded to the `target`.|
+|&nbsp;&nbsp;`exclusions`|(Only possible if `source` is used) A list of EL paths into the `source` material which should not be forwarded. Whether or not `removeSource` is used, these will be retained in their original position in the source component's options.|
+|`priority`|(Optional) a [Priority specification](Priorities.md) specifying how the priority of this distribution is to be resolved with respect to any other options distributions which are targetted at the same component in the tree as this one. Typically one will write `after:otherNamespace` to indicate that this distribution should be stronger than (should merge options on top of) another distribution whose `namespace` is `otherNamespace`, or `before:otherNamespace` to indicate that this distribution should be weaker than (have option merged on top of by) a distribution whose `namespace` is `otherNamespace`.|
```

Comment:
The parenthetical phrase "(should merge options on top of)" would benefit from "i.e." at the beginning.